### PR TITLE
fix: check for $addToSet to be not empty before passing update params

### PR DESCRIPTION
**Pull request by KevLehman**
OC-967
OC-983

### CODE REVIEW

**Review by aleksandernsilva — DISMISSED** [review]

File: .changeset/violet-beans-breathe.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+fix: check for $addToSet to be not empty before passing update
```
[the remark this answers is not in the document]
Comment (KevLehman) [reply .changeset/violet-beans-breathe.md:5]: Hmmm, think the default one is using the PR title as the actual set.

Can change it, no probs. But:
@sampaiodiego , can we make a standard then? I see some changesets using PR title, others writing a normal sentence, and others writing the setting in past 👀 at the end, the format on the changelog will be "confusing" for end users, wdyt of deciding which style should we choose?

[the remark this answers is not in the document]
Comment (sampaiodiego) [reply .changeset/violet-beans-breathe.md:5]: yeah, we need to make a decision on a standard for changesets..

I agree with Deb though, the changeset should be user friendly, it is hand rafted and separate from everything else (like it is not a commit message or the PR title) for this purpose..

btw, you can have a sneak peek of the next change log on this PR https://github.com/RocketChat/Rocket.Chat/pull/29284

[the remark this answers is not in the document]
Comment (KevLehman) [reply .changeset/violet-beans-breathe.md]: Nice. Changed to be more "user friendly". Would be good to communicate this in open too 🤗 

thx guys

**Review by murtaza98 — APPROVED** [review]

**Review by sampaiodiego — APPROVED** [review]
no tests?

**Pull request merged**